code/go/0chain.net/smartcontract/storagesc/models.go

```diff
@@ -145,6 +148,28 @@ func (sn *BlobberChallenge) addChallenge(challenge *StorageChallenge) bool {
 	return false
 }
 
+func (sn *BlobberChallenge) getSuccessCount(isSuccess bool,
+	now common.Timestamp, lastNDays int64, cct time.Duration) (count int) {
+	fromTimestamp := common.Timestamp(common.ToTime(now).Add(time.Duration(lastNDays) * time.Hour * -24).Unix())
+
+	if sn.LatestCompletedChallenge.Created < fromTimestamp {
+		return
+	}
+
+	for i := len(sn.Challenges) - 1; i >= 0; i++ {
+		if sn.Challenges[i].Created < fromTimestamp {
+			return
+		}
+		success, failure := sn.Challenges[i].getStats()
+		pass, _, _ := sn.Challenges[i].isChallengePassed(success, failure, cct, now)
+		if pass == isSuccess {
```

Comment:
Is it possible the `isSuccess` false? otherwise, we don't need this param I think.